hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -459,6 +459,7 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
     this.omNodeDetails = omhaNodeDetails.getLocalNodeDetails();
 
     omStorage = new OMStorage(conf);
+    omStorage.setOmNodeId(omNodeDetails.getNodeId(), true);
```

Comment:
Do we need this line and the `persistToDisk` flag? The node ID should have been persisted during init, so the constructor can just load it. The other fields in `OMStorage` don't have an extra `persistToDisk` boolean parameter.